clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -1923,9 +1922,9 @@ private void checkRequest(AbstractRequest req) {
         // Check for equality of the ByteBuffer only if indicated (it is likely to fail if any of the fields
         // in the request is a HashMap with multiple elements since ordering of the elements may vary)
         try {
-            ByteBuffer serializedBytes = req.serialize();
+            ByteBuffer serializedBytes = req.serializeToByteBuffer();
```

Comment:
If `req.serialize()` returned `ByteBufferAccessor`, we could use that here.  The benefit is that we could just use `parseRequest` that takes `Readable`.

Reply:
Done